gensim/models/fasttext.py

```diff
@@ -1216,15 +1216,15 @@ def _load_fasttext_format(model_file, encoding='utf-8', full_model=True):
         window=m.ws,
         iter=m.epoch,
         negative=m.neg,
-        hs=(m.loss == 1),
-        sg=(m.model == 2),
+        hs=int(m.loss == 2),
```

Comment:
Why this change from `1` to `2`? Was there a bug?

Reply:
No, my mistake I took the sequence of "enums" from help instead of cpp code.

Good catch. Corrected!